doc/api_reference.rst

```diff
@@ -200,6 +201,26 @@ API Reference
             `Pytest Command-Line Flags
             <https://docs.pytest.org/en/stable/reference/reference.html#command-line-flags>`_
 
+    * ``DISCOVERY_EXTRA_ARGS``
+
+        List of extra arguments to pass on the :term:`Pytest` command line for
+        the test discovery::
+
+            pytest_discover_tests(
+                ...
+                DISCOVERY_EXTRA_ARGS "--capture=no"
+            )
```

Comment:
Thanks for adding API docs! I would recommend showcasing a real use case here, as `--capture=no` is not really relevant for test discovery.